source/release-notes.txt

```diff
@@ -10,6 +10,36 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.1
+----------------
+
+*Released January 15, 2024*
+
+New Features:
+
+- Adds welcome page (:issue:`COMPASS-8657`)
+- Adds onFailToLoadConnections callback (:issue:`COMPASS-8657`)
```

Comment:
this two features are part of CompassWeb and currently we do not include them in the release notes. I know this came from GH release notes, however can we drop them?

Reply:
Removed.